src/Commands/MakeEnumCommand.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Regnerisch\LaravelBeyond\Commands;
+
+use Illuminate\Console\Command;
+use Regnerisch\LaravelBeyond\Resolvers\DomainNameSchemaResolver;
+
+class MakeEnumCommand extends Command
+{
+    protected $signature = 'beyond:make:enum {name}';
+
+    protected $description = 'Make a new enum type';
+
+    public function handle()
+    {
+        try {
+            $name = $this->argument('name');
```

Comment:
@regnerisch maybe add an option `backed` which automatically creates a backed enum?

Reply:
@alexgaal what should be the difference? Will the enum.stub not be the same?

Reply:
It actually would be. Let's not implement it then.